src/golomb.h

```diff
@@ -304,17 +301,16 @@ class decoder
     {
         output_buffer = 0u;
         digits        = initial_digits;
-        state         = scan_zeros;
+        state         = scan_state::scan_zeros;
         
         return output;
     }
 };
 
-template< typename OutputValueT,
-          typename InputIt,
+template< std::integral OutputValueT,
+          pg::golomb::detail::unsigned_integral_input_iterator InputIt,
```

Comment:
`pg::golomb` can be removed since the template resides in this namespace.

Reply:
Done